website/blog/2022-04-05-when-backend-devs-spark-joy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "From the Slack Archives: When Backend Devs Spark Joy for Data Folks"
+description: "An overview of an interesting community slack thread about the relationship between data folks and backend developers"
+slug: when-backend-devs-spark-joy
+
+authors: [kira_furuichi]
+
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2022-04-05
+is_featured: true
+---
+
+# From the Slack Archives: When Backend Devs Spark Joy for Data Folks
```

Comment:
I'll need to you remove this H1. The `title:` tag up top actually populated the page as the meta title & H1. So, in the current format, we would have two H1s.

I should have noted that before, sorry about that.

Reply:
Sure will do, thanks for calling that out! Just wanted to note it's like 50/50 on existing blog posts having that also as an H1 if we wanted to go back and optimize those later